pkgs/top-level/python-aliases.nix

```diff
@@ -178,6 +178,7 @@ mapAliases ({
   graphite_beacon = throw "graphite_beacon was removed, because it is no longer maintained"; # added 2022-07-09
   grappelli_safe = grappelli-safe; # added 2023-10-08
   grpc_google_iam_v1 = grpc-google-iam-v1; # added 2021-08-21
+  gtimelog = throw "gtimelog has been promoted to a top-level attribute"; # added 2023-10-10
```

Comment:
```suggestion
  inherit (pkgs) gtimelog;
```
this should work

Reply:
Changed, thanks. I have to admit I'm not entirely sure how to test that it actually works. Hopefully some automatic check will complain if it does not :)